### PR TITLE
Assignment1 Folder Added

### PR DESCRIPTION
**Pull request by roshnidurugwar**

### CODE REVIEW

**Review by Oshin-250197 — APPROVED** [review]

**Review by SamikshaWani — APPROVED** [review]

**Review by SamikshaWani — APPROVED** [review]

**Review by arshad05786 — APPROVED** [review]

**Review by yogesh141293 — APPROVED** [review]

File: Roshni/html/Assignment1/Question1.html [hunk 1]
```diff
@@ -0,0 +1,10 @@
+<!DOCTYPE html>
+<head>
+    <title>assignment1</title>
+</head>
+<body>
+    <h1>HTML</h1>
+    <h4><b>Introduction</b></h4>
+    <p>HTML is a standard markup language for Web Pages.</p>
+</body>
+</html>
```
Comment (JafarPathan) [inline Roshni/html/Assignment1/Question1.html:10]: add newline at end of file

File: Roshni/html/Assignment1/Question2.html [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+<body>
+    <h1>HTML</h1>
+    <h3><b>Introduction</b></h3>
+    <p>HTML is a standard markup language for Web Pages.</p>
+    <h3>HTML Editor</h3>
+    <h5>Notepad or TextEdit. </h5>
+    <p>Web pages can be created and modified by using professional HTML editors.<br>
+        However, for learning HTML,we recommend a simple text editor like Notepad (PC) or Textedit.(Mac).</p>
+</body>
+</html>
```
Comment (JafarPathan) [inline Roshni/html/Assignment1/Question2.html:14]: add newline at end of file

File: Roshni/html/Assignment1/Question3.html [hunk 3]
```diff
@@ -0,0 +1,15 @@
...
+    <h1>HTML</h1>
+    <h3><b>Introduction</b></h3>
+    <p>HTML is a standard markup language for Web Pages.</p>
+    <hr>
+    <h3>HTML Editor</h3>
+    <h5>Notepad or TextEdit. </h5>
+    <p>Web pages can be created and modified by using professional HTML editors.<br>
+        However, for learning HTML,we recommend a simple text editor like Notepad (PC) or Textedit.(Mac).</p>
+</body>
+</html>
```
Comment (JafarPathan) [inline Roshni/html/Assignment1/Question3.html:15]: add newline at end of file

File: Roshni/html/Assignment1/Question4.html [hunk 4]
```diff
@@ -0,0 +1,12 @@
...
+    <title>Assignment 4</title>
+</head>
+<body>
+    <h1>About Myself</h1>
+    <hr>
+    <p>I am Roshni Durugwar from Nagpur.<br><br>
+        I am student of Ambersoft Academy. 
+    </p>
+</body>
+</html>
```
Comment (JafarPathan) [inline Roshni/html/Assignment1/Question4.html:12]: add newline at end of file

File: Roshni/html/Assignment2/Question1.html [hunk 5]
```diff
@@ -0,0 +1,22 @@
...
+ <p>By some accounts, they disagreed about nearly everthing during that first meeting,but by the following year they struck a <SPAN STYLE="background-color:yellow">partnership.</SPAN> <br>
+Working from their dorm rooms,they build a search engine that used links to determine the importance of individual pages on the  <SPAN STYLE="color:blue">World Wide Web.</SPAN><br>
+they called this search engine <SPAN STYLE="background-color:red">Backrub.</SPAN></p>
+<hr>
+<h2>Initial public offering</h2>
+<p>Google's initial public offering(IPO) took place five years later,on 19<sup>th</sup> August 2004.<br>
+At that time Larry Page,Sergey Brin,and Eric Schmidt agreed to work together at Google for <del>30</del><u>20</u> years, until 2024</p>
+<h1 style="text-align: center;color: yellow;">Thank You!!</h1>
+</body>
+</html>
```
Comment (JafarPathan) [inline Roshni/html/Assignment2/Question1.html:22]: add newline at end of file

File: Roshni/html/Assignment3/Ass3.html [hunk 6]
```diff
@@ -0,0 +1,63 @@
...
+                   <li>Durugwar</li>               
+                </ul>  
+                </li>     
+            <li>MI-MEAN-9
+                <ul type=disc>
+                    <li>Lucky</li>
+                    <li>Vaishnavi</li>
+                </ul>  
+                </li>
+
```
Comment (JafarPathan) [inline Roshni/html/Assignment3/Ass3.html:40]: remove unwanted lines ans space

File: Roshni/html/Assignment3/Ass3.html [hunk 7]
```diff
@@ -0,0 +1,63 @@
...
+                <li>Barsu</li>
+                <li>Sita</li>
+            </ul>
+            </li>
+        </ol>
+        </li>
+    </ol>
+
+</body>
+</html>
```
Comment (JafarPathan) [inline Roshni/html/Assignment3/Ass3.html:63]: add newline at end of file

File: Roshni/html/Assignment3/ContactPage.html [hunk 8]
```diff
@@ -0,0 +1,63 @@
...
+            This is contact Page.This is contact Page.This is contact Page.This is contact Page.
+            This is contact Page.This is contact Page.This is contact Page.This is contact Page.
+            This is contact Page.This is contact Page.This is contact Page.This is contact Page.
+            This is contact Page.This is contact Page.This is contact Page.This is contact Page.
+            This is contact Page.This is contact Page.This is contact Page.This is contact Page.
+            
+        </pre></div>
+        </div>   
+</body>
+</html>
```
Comment (JafarPathan) [inline Roshni/html/Assignment3/ContactPage.html:63]: add newline at end of file

File: Roshni/html/Assignment3/Home.html [hunk 9]
```diff
@@ -0,0 +1,74 @@
...
+    This is Home Page Subheading.This is Home Page Subheading.
+    This is Home Page Subheading.This is Home Page Subheading.
+    This is Home Page Subheading.This is Home Page Subheading.
+    This is Home Page Subheading.This is Home Page Subheading.
+    This is Home Page Subheading.This is Home Page Subheading.
+    This is Home Page Subheading.This is Home Page Subheading.
+</p>
+</div>
+</body>
+</html>
```
Comment (JafarPathan) [inline Roshni/html/Assignment3/Home.html:74]: add newline at end of file

File: Roshni/html/Assignment3/MeanPage.html [hunk 10]
```diff
@@ -0,0 +1,45 @@
...
+            <li>Kapse</li>
+        </ul>
+<ol type="i">
+    <li>Sangita</li>
+    <li>Prakash</li>
+    <li>Durugwar</li>
+</ol>
+    </body>
+
+</html>
```
Comment (JafarPathan) [inline Roshni/html/Assignment3/MeanPage.html:45]: add newline at end of file

File: Roshni/html/Assignment4/Question1.html [hunk 11]
```diff
@@ -0,0 +1,50 @@
...
+            </tr>
+            <tr>
+                <th>Females</th>
+                <td>1.7</td>
+                <td>0.002</td>
+                <td>43%</td>
+            </tr>  
+    </table>    
+</body>
+</html>
```
Comment (JafarPathan) [inline Roshni/html/Assignment4/Question1.html:50]: add newline at end of file

File: Roshni/html/Assignment4/Question2.html [hunk 12]
```diff
@@ -0,0 +1,95 @@
...
+        <tr class="demo2" >
+        <th>4:30</th>
+        <td>&nbsp;</td>
+        <td>&nbsp;</td>
+        <td>&nbsp;</td>
+    </tr>
+    </body>
+    </table>
+    </body>
+    </html>
```
Comment (JafarPathan) [inline Roshni/html/Assignment4/Question2.html:95]: add newline at end of file

File: Roshni/html/Assignment4/Question3.html [hunk 13]
```diff
@@ -0,0 +1,89 @@
...
+        <tr>
+            <td>Wednesday</td>
+            <td class="b">8:00am</td>
+            <td class="d">12:00pm</td>
+            <td>XSL Formatting Objects</td>
+        </tr>
+    </table>
+
+</body>
+</html>
```
Comment (JafarPathan) [inline Roshni/html/Assignment4/Question3.html:89]: add newline at end of file

File: Roshni/html/Assignment2/Question1.html (hunk 5 above)
Comment (JafarPathan) [inline Roshni/html/Assignment2/Question1.html:22]: check indentation

**Review by JafarPathan — COMMENTED** [review]
For all Files 
1. remove all unwanted spaces and lines
2. do proper indentation (4 spaces )
3. add new line at end of file

File: Roshni/html/Assignment1/Question1.html (hunk 1 above)
Comment (roshnidurugwar) [reply Roshni/html/Assignment1/Question1.html:10]: yes sir

**Pull request merged**